packages/store/addon/-private/system/core-store.ts

```diff
@@ -3620,88 +3618,6 @@ function _commit(adapter, store, operation, snapshot) {
   );
 }
 
-/**
- *
- * @param store
- * @param cache modelFactoryCache
- * @param normalizedModelName already normalized modelName
- * @return {*}
- */
-function getModelFactory(store, cache, normalizedModelName) {
```

Comment:
This function and the others that were deleted in this file are not used at all so they were caught by `@typescript-eslint/no-unused-vars`.

Reply:
Another PR also deletes them, happy to rebase it though :D